packages/vite/src/plugins/plugin.ts

```diff
@@ -15,7 +15,7 @@ import { existsSync, readdirSync } from 'fs';
 import { calculateHashForCreateNodes } from '@nx/devkit/src/utils/calculate-hash-for-create-nodes';
 import { projectGraphCacheDirectory } from 'nx/src/utils/cache-directory';
 import { getLockFileName } from '@nx/js';
-import { loadViteDynamicImport } from '../utils/executor-utils';
+import { resolveConfig } from 'vite';
```

Comment:
i think we want to import it dynamically? so that we don't get the warning? @jaysoo wdyt?

Reply:
Yeah, it needs to be dynamic so we support Vite ESM, otherwise it'll break in the future (plus I think we may get a warning if going through CJS).